source/enc_ctx.c

```diff
@@ -95,6 +95,9 @@ int aws_cryptosdk_enc_ctx_serialize(
         }
         const struct aws_string *key   = (const struct aws_string *)elem.key;
         const struct aws_string *value = (const struct aws_string *)elem.value;
+        // This should have already been assured by aws_aws_cryptosdk_enc_ctx_size()
```

Comment:
What test for something previously assured?  Is there potential for a tool error that concerns you?

Reply:
This is on the actual code, and protects against future changes to the `aws_aws_cryptosdk_enc_ctx_size()` function